lib/rules/comma-dangle.js

```diff
@@ -12,7 +12,9 @@
 //------------------------------------------------------------------------------
 
 module.exports = function (context) {
-    var forbidDanglingComma = context.options[0] !== "always";
+    var allowDangle = context.options[0] || "never";
```

Comment:
You're changing the default behavior here, which isn't necessary to add the new option. Please revert.